salt/journal-cms/init.sls

```diff
@@ -319,19 +314,11 @@ journal-cms-{{ process }}-service:
             - aws-credentials-cli
 {% endfor %}
 
-
-# disabled until ubr develop -> master
-#restore-legacy-files:
-#    cmd.run:
-#        - cwd: /opt/ubr
-#        - name: |
-#            set -e
-#            source install.sh
-#            python -m ubr.main /etc/ubr/restore-only/ restore file journal-cms--platform.sh "mysql-database.legacy_cms tar-gzipped./scripts/legacy_cms_files/**"
-#            touch /root/.legacy-restored.flag
-#        - require:
-#            - file: journal-cms-backups
-#        - unless:
-#            - test -e /root/.legacy-restored.flag
-
-
+restore-legacy-files:
+    cmd.script:
+        - name: restore-legacy-script
+        - source: salt://journal-cms/scripts/restore-legacy.sh
```

Comment:
we may want to run this state only in `end2end` and `prod`, but not in `ci`. It's only needed to run all migrations and `ci` doesn't